intranet/apps/dataimport/management/commands/import_users.py

```diff
@@ -0,0 +1,78 @@
+import json
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+
+class Command(BaseCommand):
+    help = "Import Fake People"
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument("data_fname")
+
+    def handle(self, *args, **kwargs):
+        try:
+            with open(kwargs["data_fname"]) as f:
+                data = json.load(f)
```

Comment:
```suggestion
            with open(kwargs["data_fname"]) as f_obj:
                data = json.load(f_obj)
```

I usually use `f_obj` instead of `f` because linters don't like single-character variable names -- and neither do I.